dev/change_version.py

```diff
@@ -302,6 +303,24 @@ def change_project_number_Doxyfile(file, new_version):
     f.write(changed_str)
     f.close()
 
+def change_project_number_readme(file, new_version):
+    changed_str = ""
+
+    markers = ["<Reference", "<HintPath>"]
+    #                             0   .  15etc
+    version_regex = re.compile(r'[0-9]\.[0-9]+\.[0-9]+')
```

Comment:
That will match versions with *exactly* three numbers in the version - that is, it will *not* match two-number versions like `0.16`. Is that OK?

Reply:
I don't think we ever use 2-section version, release manager guide instructs to use 3-section version explicitly and in all examples. So it should be fine

Reply:
In general, probably not. But for this part of the documentation, we have only have 2 strings, 0.xx.xx and 0.xx.xx.0.